src/content/peerconnection/audio/index.html

```diff
@@ -32,50 +32,66 @@
 
-    <div id="audio">
-      <div>
-        <div class="label">Local audio:</div><audio id="audio1" autoplay controls muted></audio>
-      </div>
-      <div>
-        <div class="label">Remote audio:</div><audio id="audio2" autoplay controls></audio>
-      </div>
+  <div id="audio">
+    <div>
+      <div class="label">Local audio:</div><audio id="audio1" autoplay controls muted></audio>
     </div>
-
-    <div id="buttons">
-      <select id="codec">
-        <!-- Codec values are matched with how they appear in the SDP.
-        For instance, opus matches opus/48000/2 in Chrome, and ISAC/16000
-        matches 16K iSAC (but not 32K iSAC). -->
-        <option value="opus">Opus</option>
-        <option value="ISAC">iSAC 16K</option>
-        <option value="G722">G722</option>
-        <option value="PCMU">PCMU</option>
-      </select>
-      <button id="callButton">Call</button>
-      <button id="hangupButton">Hang Up</button>
-    </div>
-    <div class="graph-container" id="bitrateGraph">
-      <div>Bitrate</div>
-      <canvas id="bitrateCanvas"></canvas>
+    <div>
+      <div class="label">Remote audio:</div><audio id="audio2" autoplay controls></audio>
     </div>
-    <div class="graph-container" id="packetGraph">
-      <div>Packets sent per second</div>
-      <canvas id="packetCanvas"></canvas>
-    </div>
-
-    <a href="https://github.com/webrtc/samples/tree/gh-pages/src/content/peerconnection/audio" title="View source for this page on GitHub" id="viewSource">View source on GitHub</a>
+  </div>
 
+  <div id="buttons">
+    <select id="codec">
+      <!-- Codec values are matched with how they appear in the SDP.
+      For instance, opus matches opus/48000/2 in Chrome, and ISAC/16000
+      matches 16K iSAC (but not 32K iSAC). -->
+      <option value="opus">Opus</option>
+      <option value="ISAC">iSAC 16K</option>
+      <option value="G722">G722</option>
+      <option value="PCMU">PCMU</option>
+    </select>
+    <button id="callButton">Call</button>
+    <button id="hangupButton">Hang Up</button>
+  </div>
+  <div class="graph-container" id="bitrateGraph">
+    <div>Bitrate</div>
+    <canvas id="bitrateCanvas"></canvas>
   </div>
+  <div class="graph-container" id="packetGraph">
+    <div>Packets sent per second</div>
+    <canvas id="packetCanvas"></canvas>
+  </div>
+
+  <a href="https://github.com/webrtc/samples/tree/gh-pages/src/content/peerconnection/audio" title="View source for this page on GitHub" id="viewSource">View source on GitHub</a>
 
-  <script src="https://webrtc.github.io/adapter/adapter-latest.js"></script>
-  <script src="../../../js/common.js"></script>
-  <script src="js/main.js"></script>
-  <script src="../../../js/third_party/graph.js"></script>
+</div>
+<hr>
+<table>
+  <caption>Bitrate and Packes sent per second - approximate results in Chrome, Opera, Firefox, Safari</caption>
```

Comment:
If the numbers below have been measured manually in the browsers mentioned, please include the version numbers of the browsers in case things change.